common/src/main/java/com/microsoft/identity/common/internal/util/PackageUtils.java

```diff
@@ -145,6 +148,7 @@ public static final String verifySignatureHash(final @NonNull List<X509Certifica
             hashListStringBuilder.append(signatureHash);
             hashListStringBuilder.append(',');
 
+            //The next of the iterator should be overridden to get the signatureHash.
```

Comment:
I don't follow this comment

Reply:
The next() functionality of the iterator should be overridden to retrieve the signatureHash of the entry, for the purpose of comparison in the following code steps.